.ci/jenkins/Jenkinsfile.prod.nightly

```
@@ -72,7 +76,7 @@ pipeline {
         stage('Generate Properties Files') {
             steps {
                 script {
-                    if(env.ALREADY_BUILT_PROJECTS?.trim()) {
+                    if(PME_EXECUTION_RESULT?.size()) {
```

```suggestion
                    if(PME_EXECUTION_RESULT) {
```
should be enough there or ?

---

well if map is empty, `if(PME_EXECUTION_RESULT)` will consider false